common_types/src/datum.rs

```diff
@@ -269,43 +269,43 @@ impl TryFrom<u8> for DatumKind {
 impl From<DatumKind> for DataTypePb {
     fn from(kind: DatumKind) -> Self {
         match kind {
-            DatumKind::Null => Self::NULL,
-            DatumKind::Timestamp => Self::TIMESTAMP,
-            DatumKind::Double => Self::DOUBLE,
-            DatumKind::Float => Self::FLOAT,
-            DatumKind::Varbinary => Self::VARBINARY,
-            DatumKind::String => Self::STRING,
-            DatumKind::UInt64 => Self::UINT64,
-            DatumKind::UInt32 => Self::UINT32,
-            DatumKind::UInt16 => Self::UINT16,
-            DatumKind::UInt8 => Self::UINT8,
-            DatumKind::Int64 => Self::INT64,
-            DatumKind::Int32 => Self::INT32,
-            DatumKind::Int16 => Self::INT16,
-            DatumKind::Int8 => Self::INT8,
-            DatumKind::Boolean => Self::BOOL,
+            DatumKind::Null => Self::Null,
```

Comment:
Upper case in pb, and Mixed case after compiling. It seems a bit strange...?

Reply:
It is indeed strange, but I prefer this naming style, which is uniform with the enum naming in the Rust.

Reply:
Ok, indeed I have felt strange before, because upper case usually used by constant in rust.